debezium-core/src/main/java/io/debezium/schema/SchemaChangeEvent.java

```diff
@@ -63,6 +63,9 @@ public SchemaChangeEvent(Map<String, ?> partition, Map<String, ?> offset, Struct
             case DROP:
                 tables.forEach(tableChanges::drop);
                 break;
+            case TRUNCATE:
+                tables.forEach(tableChanges::truncate);
+                break;
```

Comment:
Why would a `TRUNCATE` be part of the schema change event types?  In essence, all a `TRUNCATE` consists of is a mass `DELETE` in a database-efficient way.  Is there a reason why this is here?

Reply:
Needed it for my first attempts, but seems it's not needed after all. Removing.